src/fluree/db/json_ld/transact.cljc

```diff
@@ -183,7 +184,7 @@
           {:keys [last-pid last-sid]} @iri->sid
 
           db-after (-> db-before
-                       (update :txns (fnil conj []) [txn-id author-did])
+                       (update :txns (fnil conj []) [txn-id author-did annotation])
```

Comment:
How is this `:txns` entry used?

Reply:
This is a vector of per-stage txn metadata that gets accumulated at every stage. During the commit it gets processed into commit data and removed from the db.